web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -1,5 +1,347 @@
+
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    opacity: 0.4;
+  }
+  :-ms-input-placeholder {
+    opacity: 0.4;
+  }
+`;
+
+const MyPageTitle = styled.span`
+  text-align: left;
+  font-size: 3vh;
+  font-weight: 900;
+  margin-bottom: 2vh;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const FontText = styled.span`
+  text-align: left;
+  font-size: 2vh;
+  font-weight: 700;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const ChangeButton = styled.button`
+  border: none;
+  min-width: 70px;
+  width: 10vw;
+  height: 100%;
+  padding: 1.9vh;
+  margin-left: 1vh;
+  font-size: 1.4vh;
+  font-weight: 700;
+  color: ${(props) => props.theme.colors.text};
+  background-color: ${(props) => props.theme.colors.button};
+  border-radius: ${(props) => props.theme.borderRadius};
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.colors.btnhover};
+  }
+`;
+
+const UserAgree = styled.input`
+appearance: none;
+border: 1.5px solid gainsboro;
+border-radius: 0.35rem;
+width: 1.5rem;
+height: 1.5rem;
+cursor: pointer;
+
+&:checked {
+  border-color: transparent;
+  background-image: url("data:image/svg+xml,%3csvg viewBox='0 0 16 16' fill='white' xmlns='http://www.w3.org/2000/svg'%3e%3cpath d='M5.707 7.293a1 1 0 0 0-1.414 1.414l2 2a1 1 0 0 0 1.414 0l4-4a1 1 0 0 0-1.414-1.414L7 8.586 5.707 7.293z'/%3e%3c/svg%3e");
+  background-size: 100% 100%;
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-color: limegreen;
+`;
+
+const CompleteButton = styled.button`
+  margin-top: 1vh;
+  padding: 2vh;
+  padding-left: 8vw;
+  padding-right: 8vw;
+  font-size: 2vh;
+  font-weight: 700;
+  color: white;
+  background-color: ${(props) => (props.disabled ? props.theme.colors.prhover : props.theme.colors.primary)};
+  border: none;
+  border-radius: ${(props) => props.theme.borderRadius};
+  cursor: ${(props) => (props.disabled ? 'auto' : 'pointer')};
+
+  &:hover {
+    background-color: ${(props) => props.theme.colors.prhover};
+  }
+`;
+
+type State = {
+  Email: string;
+  Password: string;
+  Password2: string;
+  Name: string;
+  PhNumber: string;
+  Key: string;
+  Email_Auth: boolean;
+  Password_Overlap: boolean;
+  Key_Request: boolean;
+  Key_Auth: boolean;
+  AgreeService: boolean;
+  AgreeInfor: boolean;
+};
+
+type Action =
+  | { type: 'SET_EMAIL'; payload: string }
+  | { type: 'SET_PASSWORD'; payload: string }
+  | { type: 'SET_CONFIRM_PASSWORD'; payload: string }
+  | { type: 'SET_NAME'; payload: string }
+  | { type: 'SET_PHONE_NUMBER'; payload: string }
+  | { type: 'SET_AUTH_KEY'; payload: string }
+  | { type: 'AUTH_EMAIL'; payload: boolean }
+  | { type: 'OVERLAP_PASSWORD'; payload: boolean }
+  | { type: 'REQUEST_KEY'; payload: boolean }
+  | { type: 'AUTH_KEY'; payload: boolean }
+  | { type: 'AGREE_SERVICE'; payload: boolean }
+  | { type: 'AGREE_INFORMATION'; payload: boolean };
+
+const initalState = {
+  Email: '',
+  Password: '',
+  Password2: '',
```

Comment:
Please don't use numbers with variables. I've seen this kind of names on some pretty popular projects. But this is just way too bad practice. Each variable should be able to represent itself.

Maybe `confirmPassword` should be fine. And for variables in `initialState` why all the keys are in **PascalCase**? variables (not constants) should be **camelCase**.

Reply:
OK! it is fixed in RegisterPage. but ProfilePage will be fixed after transferring to another branch! it will not fixed in this branch, can you understand it?